package.json

```diff
@@ -60,8 +60,9 @@
     "@hapi/hoek": "^9.0.4",
     "@hapi/inert": "^6.0.3",
     "@vrbo/steerage": "^11.0.1",
-    "hapi-pino": "^8.5.0",
```

Comment:
This isn't in this change, but likely and oversight from a prior PR; but the runtime deps should likely be moved to `peerDependencies` and `devDependencies`.  The ones I'm thinking of are `@hapi/hoek`, `hapi-pino`, `pino-pretty`, `joi` and likely a few others.

`@hapi/hapi` is in `dependencies` and `peerDependencies`; that doesn't make sense and isn't the intent of how we want to manage these.